src/analyses/variable-sensitivity/abstract_value.h

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************\
+
+ Module: Analyses Variable Sensitivity
+
+ Author: Thomas Kiley, thomas.kiley@diffblue.com
+
+\*******************************************************************/
+#ifndef CPROVER_ANALYSES_VARIABLE_SENSITIVITY_ABSTRACT_VALUE_H
+#define CPROVER_ANALYSES_VARIABLE_SENSITIVITY_ABSTRACT_VALUE_H
+
+#include <analyses/variable-sensitivity/abstract_object.h>
+
+
+class abstract_valuet:public abstract_objectt
```

Comment:
What's the difference between an abstract value and and abstract object? I would have expected to have only one of those, with either name.